js/webpack_get_defaults.js

```diff
@@ -240,6 +241,7 @@ module.exports = function (basePath) {
         use: loaders,
       })
     }
+    plugins.push(new Webpack.DefinePlugin({'process.env': {NODE_ENV: '"production"'}}))
```

Comment:
I'm not sure you need the double quotes around "production"; even worse, you may introduce a bug.

Reply:
This is the way it is written in the vuejs doc : https://fr.vuejs.org/v2/guide/deployment.html
It works well in the tests I made => make dist replaces all the `process.env.NODE_ENV` by `false`.
An alternative seems to be: `JSON.stringify(process.env.NODE_ENV)`

Reply:
You're right, interseting btw.